modules/infra/variables.tf

```diff
@@ -226,15 +232,17 @@ variable "default_node_groups" {
 variable "additional_node_groups" {
   description = "Additional EKS managed node groups definition."
   type = map(object({
-    ami                   = optional(string, null)
-    bootstrap_extra_args  = optional(string, "")
-    instance_types        = list(string)
-    spot                  = optional(bool, false)
-    min_per_az            = number
-    max_per_az            = number
-    desired_per_az        = number
-    availability_zone_ids = list(string)
-    labels                = map(string)
+    ami                        = optional(string, null)
+    bootstrap_extra_args       = optional(string, "")
+    instance_types             = list(string)
+    spot                       = optional(bool, false)
+    min_per_az                 = number
+    max_per_az                 = number
+    max_unavailable_percentage = optional(number)
+    max_unavailable            = optional(number)
```

Comment:
I think we probably meant to set this one?

Reply:
Whoops, yep!  Just updated